mongo/options/gridfsoptions.go

```diff
@@ -25,56 +25,66 @@ var DefaultChunkSize int32 = 255 * 1024 // 255 KiB
 // DefaultRevision is the default revision number for a download by name operation.
 var DefaultRevision int32 = -1
 
-// BucketOptions represents all possible options to configure a GridFS bucket.
+// BucketOptions represents options that can be used to configure GridFS bucket.
 type BucketOptions struct {
-	Name           *string                    // The bucket name. Defaults to "fs".
-	ChunkSizeBytes *int32                     // The chunk size in bytes. Defaults to 255KB.
-	WriteConcern   *writeconcern.WriteConcern // The write concern for the bucket. Defaults to the write concern of the database.
-	ReadConcern    *readconcern.ReadConcern   // The read concern for the bucket. Defaults to the read concern of the database.
-	ReadPreference *readpref.ReadPref         // The read preference for the bucket. Defaults to the read preference of the database.
+	// The name of the bucket. The default value is "fs".
+	Name *string
+
+	// The number of bytes in each chunk in the bucket. The default value is DefaultChunkSize (255 KiB).
```

Comment:
Since you're referencing DefaultChunkSize here, move the value of that variable from this doc string to the doc string for the variable.

Reply:
I changed the `DefaultChunkSize` doc comment to include the value and changed the option's doc comment to not reference `DefaultChunkSize` at all. This is because the options structs and the constants will be in different places in the documentation and there is no way to link them using GoDoc. It didn't seem like good UX to reference a constant that the user has to find manually.